docs/azure/sdk/resource-management.md

```diff
@@ -132,19 +158,19 @@ Imagine that our company requires all virtual machines to be tagged with the own
 var armClient = new ArmClient(new DefaultAzureCredential());
 
 // Next we get a resource group object
-// ResourceGroup is a [Resource] object from above
-Subscription subscription = await armClient.GetDefaultSubscriptionAsync();
-ResourceGroup resourceGroup = await subscription.GetResourceGroups().GetAsync("myRgName");
+// ResourceGroup is a [ResourceName] object from above
+SubscriptionResource subscription = await armClient.GetDefaultSubscriptionAsync();
+ResourceGroupResource resourceGroup = (await subscription.GetResourceGroups().GetAsync("myRgName").Value);
 
 // Next we get the collection for the virtual machines
-// vmCollection is a [Resource]Collection object from above
+// vmCollection is a [ResourceName]Collection object from above
 VirtualMachineCollection vmCollection = resourceGroup.GetVirtualMachines();
 
 // Next we loop over all vms in the collection
-// Each vm is a [Resource] object from above
-await foreach(VirtualMachine vm in vmCollection.GetAllAsync())
+// Each vm is a [ResourceName] object from above
+await foreach(VirtualMachineResource vm in vmCollection.GetAllAsync())
 {
-    // We access the [Resource]Data properties from vm.Data
+    // We access the [ResourceName]Data properties from vm.Data
     if(!vm.Data.Tags.ContainsKey("owner"))
     {
         // We can also access all operations from vm since it is already scoped for us
```

Comment:
The below line is using the old API, which should be 
await vm.AddTagAsync("owner", GetOwner());